libraries/ui-library/src/functional-components/form-control/form-control.scss

```diff
@@ -24,7 +24,8 @@
 
   .form-control__label__required::after {
     color: var(--six-color-danger-800);
-    content: '*';
+    content: ' *';
+    font-size: 110%;
```

Comment:
* Changing the font size might increase the height of the label (happens for inputs of size large). `scale()` can be used as an alternative, which doesn't have this problem
* Instead of a space in front of the the `*`, I'd rather use a left margin with a spacing design token.

Suggestion:

```css
    content: "*";
    transform: scale(1.1);
    margin-left: var(--six-spacing-xxx-small);
    position: absolute;
```